app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, old_file, path, *_ = command.split()
+
+    if command_name == "mv" and old_file != path:
+        last_slash_index = path.rfind("/")
```

Comment:
Do not use any specific directory delimiter in your code, your code should be platform-agnostic. Use methods from the `os` module to obtain the path and filename.